forest/cli/src/cli/wallet_cmd.rs

```diff
@@ -23,9 +23,33 @@ use fvm_shared::{
 };
 use num::BigInt;
 use rpassword::read_password;
+use rust_decimal::prelude::*;
+use rust_decimal_macros::dec;
 
 use super::{handle_rpc_err, Config};
 
+const NUM_SIGNIFICANT_DIGITS: u32 = 4;
+
+#[allow(clippy::enum_variant_names)]
+enum FormattingMode {
+    /// mode to show data in `FIL` units
+    /// in full accuracy
+    /// E.g. 0.0000002367798 `FIL` instead of ~0 `FIL`
```

Comment:
I don't think it makes sense to say "X instead of Y" in this context.

Let's simplify the documentation to just a single, illustrative example of the output of each mode.

Reply:
Done